File: backend/model/db_mongo.py
import pymongo

MONGO_SERVER = 'mongodb+srv://pwith:pwith1234@cluster0.ezfau5x.mongodb.net/'

def conn_mongodb() :
    try:
        mongo_conn = pymongo.MongoClient(MONGO_SERVER)
        mongo_conn.admin.command('ismaster')
    except:
        print("pymongo exception")
        mongo_conn = pymongo.MongoClient(MONGO_SERVER)
    finally:
        pwith_db = mongo_conn.pwith_db
    return pwith_db

File: backend/model/db_mongo.js
// import mongoose from "mongoose";

// const MONGO_SERVER =
//   "mongodb+srv://pwith:pwith1234@cluster0.ezfau5x.mongodb.net/";

// const db = () => {
//   function connect() {
//     mongoose.connect(MONGO_SERVER, { dbName: "pwith_db" });
//   }
//   connect();
//   const conn = mongoose.connection;
//   conn.once("open", () => {
//     console.log("Connected to mongoDB");
//   });
//   conn.on("disconnected", connect);
//   conn.on("error", console.log);
// };

// export default db;
